### PR TITLE
[new] Knowledge Base backup 

### PR DESCRIPTION
**Pull request by Rajakavitha1**
# Description

Moved [the backup article
](https://kb-managed.timescale.com/en/articles/2752581-postgresql-backups) to docs

# Links

Fixes #[insert issue link, if any]

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [x] Is the content technically accurate?
- [x] Is the content complete?
- [x] Is the content presented in a logical order?
- [x] Does the content use appropriate names for features and products?
- [x] Does the content provide relevant links to further information?

## Documentation team review checklist

- [x] Is the content free from typos?
- [x] Does the content use plain English?
- [x] Does the content contain clear sections for concepts, tasks, and references?
- [x] Are procedure and highlight tags used appropriately?
- [x] Has the index been updated appropriately?
- [x] Have any images been uploaded to the correct location, and are resolvable?
- [x] Are all links provided in reference style, and resolvable?
- [x] If the page index was updated, are redirects required
      and have they been implemented?
- [x] Have you checked the built version of this content?

### CODE REVIEW

File: mst/backup-types.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: Types of Backups
```
Comment (charislam) [inline mst/backup-types.md:2]:
```suggestion
title: Types of backups in Managed Service for TimescaleDB
```
Is should be MST throughout, right? The entire KB is MST if I remember correctly, so the references to Cloud should be out of date (from before the rename)

File: mst/backup-types.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+excerpt: Understand how Timescale Cloud backs up PostgreSQL
+product: cloud
+keywords: [backup, binary, logical]
+tags: [backup, binary, logical]
```
Comment (charislam) [inline mst/backup-types.md:6]:
```suggestion
keywords: [backup]
tags: [backup, binary backup, logical backup]
```
You could also keep binary and logical in the keywords if you prefer (keywords get show, tags are invisible and only help rank related content)

File: mst/backup-types.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+---
+# How Timescale Cloud backs up PostgreSQL?
```
Comment (charislam) [inline mst/backup-types.md:8]:
```suggestion

# Backups in Managed Service for TimescaleDB
```

File: mst/backup-types.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+Timescale Cloud databases are automatically backed-up, with full backups daily,
```
Comment (charislam) [inline mst/backup-types.md:9]:
```suggestion
Timescale Cloud databases are automatically backed up, with full backups daily,
```

File: mst/backup-types.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+and write-ahead log (WAL) recorded continuously. All backups are
+[encrypted][avien-encrypt]. Timescale Cloud uses [`pghoard`][pghoard], a PostgreSQL® backup
+daemon and restore tool to store backup data in cloud object stores. The number of
```
Comment (charislam) [inline mst/backup-types.md:12]:
```suggestion
daemon and restore tool, to store backup data in cloud object stores. The number of
```

File: mst/index.md [hunk 6]
```diff
@@ -77,5 +78,7 @@ For more information on Managed Service for TimescaleDB's terms of service, visi
 [security]: /mst/:currentVersion:/security
 [sign-up]: https://www.timescale.com/mst-signup
 [tos]: https://www.timescale.com/legal/mst-terms-of-service
+
+[backups]: /backup-types
```
Comment (charislam) [inline mst/index.md:82]:
```suggestion
[backups]: /mst/:currentVersion:/backup-types
```

**Review by charislam — CHANGES_REQUESTED** [review]

File: mst/backup-types.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+title: Types of backups in Managed Service for TimescaleDB
+excerpt: Understand how Timescale Cloud backs up PostgreSQL
+product: cloud
+keywords: [backup]
+tags: [backup, binary backup, logical backup]
+---
+
+# Backups in Managed Service for TimescaleDB
+Timescale Cloud databases are automatically backed up, with full backups daily,
+and write-ahead log (WAL) recorded continuously. All backups are
```
Comment (Loquacity) [inline mst/backup-types.md:11]:
```suggestion
and a write-ahead log (WAL) continuously recorded. All backups are
```

File: mst/backup-types.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+[encrypted][avien-encrypt]. Timescale Cloud uses [`pghoard`][pghoard], a PostgreSQL® backup
```
Comment (Loquacity) [inline mst/backup-types.md:12]:
```suggestion
[encrypted][avien-encrypt]. Timescale Cloud uses [`pghoard`][pghoard], a PostgreSQL backup
```

File: mst/backup-types.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+daemon and restore tool, to store backup data in cloud object stores. The number of
+backups stored and the retention time of the backup depends on the service plan.
+
+<highlight type="important"> The size of logical backups, and the size of the
+Timescale Cloud backup that appears on the Timescale Cloud web console differs, in
+some cases significantly. Backup sizes that appear in the Timescale Cloud web
+console are for daily backups, before encryption and compression. To view the
+size of each database, including space consumed by indexes, you can use the `\l+` command at the psql prompt. </highlight>
```
Comment (Loquacity) [inline mst/backup-types.md:20]:
```suggestion
<highlight type="important"> 
The size of logical backups, and the size of the Timescale Cloud backup that 
appears on the Timescale Cloud web console differs, in some cases 
significantly. Backup sizes that appear in the Timescale Cloud web console 
are for daily backups, before encryption and compression. To view the size 
of each database, including space consumed by indexes, you can use 
the `\l+` command at the psql prompt. 
</highlight> 
```

File: mst/backup-types.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+backups stored and the retention time of the backup depends on the service plan.
+
+<highlight type="important"> The size of logical backups, and the size of the
+Timescale Cloud backup that appears on the Timescale Cloud web console differs, in
+some cases significantly. Backup sizes that appear in the Timescale Cloud web
+console are for daily backups, before encryption and compression. To view the
+size of each database, including space consumed by indexes, you can use the `\l+` command at the psql prompt. </highlight> 
+
+The two types of backups are binary backups and logical backups. Full backups
+are version-specific binary backups, which when combined with WAL allow
```
Comment (Loquacity) [inline mst/backup-types.md:23]:
```suggestion
are version-specific binary backups which, when combined with WAL, allow
```

File: mst/backup-types.md [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+consistent recovery to a point in time (PITR). Logical backups can be
+taken using the `pg_dump` command.
```
Comment (Loquacity) [inline mst/backup-types.md:25]:
```suggestion
consistent recovery to a point in time (PITR). You can create a 
logical backup with the `pg_dump` command.
```

File: mst/backup-types.md [hunk 12]
```diff
@@ -0,0 +1,37 @@
...
+daemon and restore tool, to store backup data in cloud object stores. The number of
+backups stored and the retention time of the backup depends on the service plan.
+
+<highlight type="important"> The size of logical backups, and the size of the
+Timescale Cloud backup that appears on the Timescale Cloud web console differs, in
+some cases significantly. Backup sizes that appear in the Timescale Cloud web
+console are for daily backups, before encryption and compression. To view the
+size of each database, including space consumed by indexes, you can use the `\l+` command at the psql prompt. </highlight> 
+
+The two types of backups are binary backups and logical backups. Full backups
```
Comment (Loquacity) [inline mst/backup-types.md:22]: Can we move this section under the next heading, above the table?

File: mst/index.md [hunk 13]
```diff
@@ -17,6 +17,7 @@ management.
 *   View [service logs][viewing-service-logs]
 *   Use [VPC peering][vpc-peering]
 *   Understand [security for Timescale MST][security]
+*   Understand [backups] [backups]
```
Comment (Loquacity) [inline mst/index.md:20]:
```suggestion
*   Understand [backups][backups]
```

File: mst/backup-types.md (hunk 12 above)
Comment (Rajakavitha1) [reply mst/backup-types.md]: done!!

**Review by Loquacity — APPROVED** [review]
This looks fine, but I wonder if it's possible to put a line or two of explanation above the table. It's not immediately clear what the table is actually trying to say until you've stared at it for a little while.

File: mst/backup-types.md [hunk 14]
```diff
@@ -28,6 +28,7 @@ are version-specific binary backups which, when combined with WAL, allow
 consistent recovery to a point in time (PITR). You can create a 
 logical backup with the `pg_dump` command.
 
+This table lists the differences binary and logical backups with respect to indexes, transactions, and data.
```
Comment (Loquacity) [inline mst/backup-types.md:31]:
```suggestion
This table lists the differences between binary and logical backups when backing up indexes, transactions, and data:

```

File: mst/backup-types.md [hunk 15]
```diff
@@ -28,6 +28,7 @@ are version-specific binary backups which, when combined with WAL, allow
 consistent recovery to a point in time (PITR). You can create a 
 logical backup with the `pg_dump` command.
 
+This table lists the differences binary and logical backups with respect to indexes, transactions, and data. 
 |Type|Binary|Logical|
 |---|---|---|
```
Comment (Loquacity) [inline mst/backup-types.md:33]:
```suggestion
|-|-|-|
```

File: mst/backup-types.md [hunk 16]
```diff
@@ -28,6 +28,7 @@ are version-specific binary backups which, when combined with WAL, allow
...
 |index|contains all data from indexes| does not contain index data, it contains only queries used to recreate indexes from other data|
```
Comment (Loquacity) [inline mst/backup-types.md:34]:
```suggestion
|index|contains all data from indexes|does not contain index data, it contains only queries used to recreate indexes from other data|
```

**Comment by alexthinath** [comment]
Broader question: Is this intending to merge over the content for MST, and still have it be specific to MST? Or replicate it for Cloud? I ask because this article is only for MST functionality, although it says "Cloud" thats because its the old name for MST

**Comment by Rajakavitha1** [comment]
> Broader question: Is this intending to merge over the content for MST, and still have it be specific to MST? Or replicate it for Cloud? I ask because this article is only for MST functionality, although it says "Cloud" thats because its the old name for MST

Hi @alexthinath !!! it was a miss at my end. I just updated the product type to mst.

**Review by alexthinath — APPROVED** [review]

**Comment by Rajakavitha1** [comment]
Hi @charislam  could you please take a look at this PR and state your approval after you review the changes that I have corporate.

File: mst/backup-types.md [hunk 17]
```diff
@@ -0,0 +1,42 @@
+---
+title: Types of backups in Managed Services for TimescaleDB
+excerpt: Understand how Timescale Cloud backs up PostgreSQL
+product: cloud
+keywords: [backup]
+tags: [backup, binary backup, logical backup]
+---
+# Backups in Managed Service for TimescaleDB
```
Comment (charislam) [inline mst/backup-types.md:8]:
```suggestion

# Backups in Managed Service for TimescaleDB
```

File: mst/backup-types.md [hunk 18]
```diff
@@ -0,0 +1,42 @@
+---
+title: Types of backups in Managed Services for TimescaleDB
+excerpt: Understand how Timescale Cloud backs up PostgreSQL
```
Comment (charislam) [inline mst/backup-types.md:3]:
```suggestion
excerpt: Understand how Managed Service for TimescaleDB backs up PostgreSQL
```

File: mst/backup-types.md [hunk 19]
```diff
@@ -0,0 +1,42 @@
...
+product: cloud
```
Comment (charislam) [inline mst/backup-types.md:4]:
```suggestion
product: MST
```

**Review by charislam — APPROVED** [review]

**Pull request merged**